src/components/Create/components/pages/ProjectToken/components/CustomTokenSettings/CustomTokenSettings.tsx

```diff
@@ -227,22 +227,32 @@ export const CustomTokenSettings = () => {
 
       <Divider className="my-8" />
 
-      <>
+      <Space direction="vertical">
```

Comment:
nit: trying to move away from `Space` where possible (and most antd as we can) - `div className="flex flex-col"` does the same thing 99% of the time, without buginess of antd. For some reason, Space vertical never fills properly for me, and I always have to use a w-full anyway 🤷‍♂️

Reply:
Done, good call!